### ISSUE TITLE
The XRay Image query crashes in parallel with some processor counts.

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#999]
The XRay Image query crashes when running in parallel with some processor counts. Some processor counts work and others don't.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Eric Brugger
Original creation: 03/18/2011 06:03 pm
Original update: 03/18/2011 07:55 pm
Ticket number: 656
